bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -261,19 +279,19 @@ public static double[] hsbToXY(HSBType hsb, Gamut gamut) {
      * Transform <a href="https://en.wikipedia.org/wiki/SRGB">sRGB</a> color format to
      * <a href="https://en.wikipedia.org/wiki/HSL_and_HSV">HSV</a> based {@link HSBType}.
      *
-     * @param rgb array of three int with the RGB values in the range 0 to 255.
+     * @param rgbw array of four int with the RGBW values in the range 0 to 255.
```

Comment:
array of 3 or 4 int values representing either RGB or RGBW values in range....

Reply:
Done, thanks.